src/components/Tables/ProductsTable/ProductsTable.js

```diff
@@ -47,13 +47,20 @@ const ProductsTable = () => {
 
   let result = products;
   if (searchTerm) {
-    result = products.filter((product) =>
-      product.description.toLowerCase().includes(searchTerm.toLowerCase())
-    );
+    result = products.filter((product) => {
+      var digits = product._id.toString();
```

Comment:
not sure here Lilian, baka dapat loop siya to loop for all the fields, and not just handpicked? For example searching for UPC and/or Supplier code, or Collections or Tags